Only in lfscmd-0.5-new/src: .xml.c.swp
diff -ur lfscmd-0.5/src/main.c lfscmd-0.5-new/src/main.c
--- lfscmd-0.5/src/main.c	2002-12-08 11:47:24.000000000 -0600
+++ lfscmd-0.5-new/src/main.c	2003-01-31 20:31:15.000000000 -0600
@@ -32,7 +32,7 @@
 	char *xmlfile;
 
 	opterr=0;
-	while ((option=getopt(argc, argv, "xft-")) != -1) {
+	while ((option=getopt(argc, argv, "xftp-")) != -1) {
 		switch (option) {
 			case 'x':
 				lfsopts.executable=1;
@@ -43,6 +43,13 @@
 			case 't':
 				lfsopts.titles=1;
 				break;
+			case 'p':
+				lfsopts.prefix=1;
+				lfsopts.files=1;
+				lfs.filecount=0;
+				lfs.lastfilename=NULL;
+				lfs.step=1;
+				break;
 			case '-':
 				usage(argv[0]);
 				return(-1);
@@ -67,5 +74,5 @@
 }
 
 void usage (char *progname) {
-	fprintf(stderr, "Usage: %s [-tfx] INDEX.XML\n\n-t  Print titles (as comments).\n-f  Write commands to their own file.\n-x  Give executable permission to the resulting files.\n", progname);
+	fprintf(stderr, "Usage: %s [-tfpx] INDEX.XML\n\n-t  Print titles (as comments).\n-f  Write commands to their own file.\n-x  Give executable permission to the resulting files.\n-p  Create Sys-V style prefixes on the resulting files. (implies -f)\n\n", progname);
 }
diff -ur lfscmd-0.5/src/xml.c lfscmd-0.5-new/src/xml.c
--- lfscmd-0.5/src/xml.c	2002-12-08 11:47:24.000000000 -0600
+++ lfscmd-0.5-new/src/xml.c	2003-02-01 03:09:15.000000000 -0600
@@ -50,6 +50,10 @@
 		retval=lfscmd(doc, node);
 
 	xmlFreeDoc(doc);
+	if (1 == lfsopts.prefix) {
+		free(lfs.filename);
+		free(lfs.lastfilename);
+	}
 	return(retval);
 }
 
@@ -90,13 +94,27 @@
 
 int parse_screen (xmlDocPtr doc, xmlNodePtr node) {
 	FILE *output=stdout;
+	char *tmp;
 
 	if (NULL == lfs.filename)
 		lfs.filename="UNKNOWN";
 	if (NULL == lfs.title)
 		lfs.title="UNKNOWN";
-	if (1 == lfsopts.files)
+	if (1 == lfsopts.files) {
+		if (1 == lfsopts.prefix) {
+			if (lfs.lastfilename == NULL || strcmp (lfs.filename, lfs.lastfilename) != 0) {
+				tmp = malloc ((strlen(lfs.filename) + 5) * sizeof (char));
+				sprintf (tmp, "%03i-%s", lfs.filecount, lfs.filename);
+				lfs.filecount += lfs.step;
+				lfs.filename = realloc (lfs.filename, (strlen(tmp) + 1) * sizeof(char));
+				strcpy (lfs.filename, tmp);
+				lfs.lastfilename = realloc (lfs.filename, (strlen(tmp) + 1) * sizeof(char));
+				strcpy (lfs.lastfilename, tmp);
+				free (tmp);
+			}
+		}
 		output=openFile(lfs.filename);
+	}
 
 	print_title(output);
 
diff -ur lfscmd-0.5/src/xml.h lfscmd-0.5-new/src/xml.h
--- lfscmd-0.5/src/xml.h	2002-12-08 11:47:24.000000000 -0600
+++ lfscmd-0.5-new/src/xml.h	2003-01-31 17:36:34.000000000 -0600
@@ -4,12 +4,12 @@
 #include <libxml/tree.h>
 
 struct LFSCMD {
-	char *title, *userinput, *text, *filename;
-	int title_status;
+	char *title, *userinput, *text, *filename, *lastfilename;
+	int title_status, filecount, step;
 } lfs;
 
 struct LFSCMD_OPTIONS {
-	int executable, files, titles;
+	int executable, files, titles, prefix;
 } lfsopts;
 
 extern int parsedoc (char *xmlfile);
